Frameworks/Foundation/NSString.mm

```diff
@@ -1427,36 +1427,25 @@ - (NSString*)stringByPaddingToLength:(NSUInteger)newLength withString:(NSString*
     return mutableCopy;
 }
 
+NSString* s_percentEncodedFormat = @"%%%s%X";
+const int s_oneByte = 16;
```

Comment:
Yup, left over from testing various fixes to this API. Forgot to remove.

---
In reply to: [77075166](https://github.com/Microsoft/WinObjC/pull/946#discussion_r77075166) [](ancestors = 77075166)